Makefile

```diff
@@ -17,7 +17,7 @@ endif
 BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
 
 # Image URL to use all building/pushing image targets
-IMG_PREFIX ?= quay.io/${USER}
+IMG_PREFIX ?= quay.io/${USER}/opentelemetry-operator
```

Comment:
Maybe this should be moved into `IMG`, so the prefix remains `quay.io/${USER}`?

Reply:
IMG includes the version too, which we might not want to override while testing.

Reply:
I think I still prefer the scope of the previous `IMG_PREFIX` value :smile: